packages/react/src/components/checkbox/Checkbox.tsx

```diff
@@ -113,7 +126,7 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
           type="checkbox"
           disabled={disabled}
           checked={checked}
-          aria-describedby={errorText ? `${id}-error` : null}
+          aria-describedby={ariaDescribedby.length > 0 ? ariaDescribedby.join(' ') : null}
```

Comment:
There is a util for creating the aria-describedBy label
https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/utils/composeAriaDescribedBy.ts
Probably would be wise to use it here as well